dev/src/index.ts

```diff
@@ -1363,130 +1319,70 @@ export class Firestore {
   }
 
   /**
-   * A funnel for read-only streaming API requests, assigning a project ID where
-   * necessary within the request options.
+   * A funnel for streaming API requests, assigning a project ID where necessary
+   * within the request options.
    *
    * The stream is returned in paused state and needs to be resumed once all
    * listeners are attached.
    *
    * @private
    * @param methodName Name of the streaming Veneer API endpoint that
    * takes a request and GAX options.
+   * @param mode Whether this a unidirectional or bidirectional call.
    * @param request The Protobuf request to send.
    * @param requestTag A unique client-assigned identifier for this request.
-   * @param allowRetries Whether this is an idempotent request that can be
-   * retried.
    * @returns A Promise with the resulting read-only stream.
    */
-  readStream(
```

Comment:
Is it a breaking change?

Reply:
This is not part of of our public API (we define [our own types](https://github.com/googleapis/nodejs-firestore/blob/master/types/firestore.d.ts)). I can use an underscore prefix here, but in general the client is pretty inconsistent here.

Reply:
OK. Since it's TypeScript, we might as well make those methods `private`.

Reply:
Unfortunately, these are "module private" and used in other internal classes.